src/diagnostics.rs

```diff
@@ -275,7 +275,7 @@ pub fn report_error<'tcx, 'mir>(
     for (i, frame) in ecx.active_thread_stack().iter().enumerate() {
         trace!("-------------------");
         trace!("Frame {}", i);
-        trace!("    return: {:?}", frame.return_place.map(|p| *p));
+        trace!("    return: {:?}", frame.return_place);
```

Comment:
Should this be `*frame.return_place`?
The intention of the `map(|p| *p)` probably was to `Debug`-print a `MemPlace`, not a `MPlaceTy`. The latter are very verbose due the layout they contain.

Reply:
Makes sense, fixed